lib/pos_blockchain.go

```diff
@@ -175,6 +168,14 @@ func (bc *Blockchain) validateBlockIntegrity(desoBlock *MsgDeSoBlock) error {
 		// we've validated that all ancestors exist in the block index.
 		return RuleErrorMissingParentBlock
 	}
+	if parentBlock.CommittedStatus == COMMITTED {
```

Comment:
Do we really need this here? I assume we check elsewhere that the committed tip is an ancestor if the incoming block, which would make the check here redundant. Maybe I'm missing something?

Reply:
Interesting. But this check is incomplete, no? If the incoming block's parent is uncommitted, but its grandparent is a committed block that's NOT the committed tip, then this check won't catch it. 

Maybe I'm missing something?